crates/rspack/tests/tree-shaking/rollup-unused-var/snapshot/snap.diff

```diff
@@ -0,0 +1,11 @@
+--- expected
++++ actual
+@@ -11,7 +11,7 @@
+ });
+ var foo = "lol";
+ var bar = "wut";
+-var baz = bar || foo;
```

Comment:
align to webpack: 
![image](https://github.com/web-infra-dev/rspack/assets/17974631/9690822c-e166-47fa-8cc4-82de4a2710fb)